packages/twenty-server/src/engine/core-modules/record-transformer/services/record-input-transformer.service.ts

```diff
@@ -96,11 +96,19 @@ export class RecordInputTransformerService {
 
     const serverBlockNoteEditor = ServerBlockNoteEditor.create();
 
-    const convertedMarkdown = parsedValue.blocknote
-      ? await serverBlockNoteEditor.blocksToMarkdownLossy(
-          JSON.parse(parsedValue.blocknote),
-        )
-      : null;
+    // Patch: Handle cases where blocknote to markdown conversion fails for certain block types (custom/code blocks)
+    // Todo : This could be resolved once the server-utils library is updated with proper conversion support
+    let convertedMarkdown: string | null = null;
+
+    try {
+      convertedMarkdown = parsedValue.blocknote
+        ? await serverBlockNoteEditor.blocksToMarkdownLossy(
+            JSON.parse(parsedValue.blocknote),
+          )
+        : null;
+    } catch {
+      convertedMarkdown = parsedValue.blocknote;
```

Comment:
This comment goes beyond this PR : 

I am not sure where the markdown can be used for our users. Maybe this would need an extra eye "métier" to make sure it's fine to leave it this way for now